Priyankaben Patel/week_01/objects/js/Homework.js

```diff
@@ -0,0 +1,216 @@
+/*Part 1, Rectangle
+Given the following a rectangle object like the one below, write the following functions which accept a rectangle object as an argument:
+
+* isSquare - Returns whether the rectangle is a square or not
+* area - Returns the area of the rectangle
+* perimeter - Returns the perimeter of the rectangle
+*/
+
+const rectangle1 = {
+  length: 8,
+  width: 5
+};
+const rectangle2 = {
+  length: 5,
+  width: 5
+};
+
+
+const checkRectangle = function (rectangle) {
+  const rectLength = rectangle.length;
+  const rectWidth = rectangle.width;
+  const area = rectLength * rectWidth;
+  const perimeter = ( 2 * (rectLength + rectWidth))
+
+  console.log (`The area is ${area}`);
+  console.log (`The perimeter is ${perimeter}`);
+
+  if (rectWidth === rectLength ) {
+    console.log('This is not rectangle');
+  }
+else {
+  console.log('This is arectangle');
+}
+}
+checkRectangle(rectangle2);
+
+//Part 2, Triangle
+/*Given the following a triangle object like the one below, write the following functions which accept a triangle object as an argument:
+
+isEquilateral - Returns whether the triangle is equilateral or not
+isIsosceles - Returns whether the triangle is isosceles or not
+area - Returns the area of the Triangle
+isObtuse - Returns whether the triangle is obtuse or not */
+const triangle1 = {
+  sideA: 6,
+  sideB: 6,
+  sideC: 6,
+  angleA: 60,
+  angleB: 60,
+  angleC: 60
+};
+
+const triangle2 = {
+  sideA: 10,
+  sideB:  10,
+  sideC: 5,
+  angleA: 70,
+  angleB: 60,
+  angleC: 50
+};
+const triangle3 = {
```

Comment:
The `triangle` object should not have the `angle` properties because you should be able to calculate that from `sideA`, `sideB` and `sideC`.